packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -314,17 +312,17 @@ export interface ISpinButtonStyles {
   spinButtonWrapperTopBottom: IStyle;
 
   /**
-   * Style override when spinButton is enabled/hovered.
+   * Style override when control is enabled/hovered.
    */
   spinButtonWrapperHovered: IStyle;
 
   /**
-   * Style override when spinButton is enabled/focused.
+   * Style override when SpinButton is enabled/focused.
```

Comment:
I see that we're mostly using `control` but there are still some inconsistent references to `SpinButton` in the types.